api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/policy/specialrelationships/RelationshipChecker.java

```diff
@@ -39,16 +39,15 @@
 	protected static final String URI_INHERES_IN = NS_OBO + "RO_0000052";
 	protected static final String URI_REALIZES = NS_OBO + "BFO_0000055";
 
-	private final OntModel ontModel;
-
-	public RelationshipChecker(OntModel ontModel) {
-		this.ontModel = ontModel;
+	public RelationshipChecker() {
 	}
 
+	public abstract boolean isRelated(OntModel ontModel, List<String> fromUris, List<String> toUris);
```

Comment:
This is true, although the previous version of the class took an ontModel in the constructor and stored it as a field - and it wasn't documented as to what model is being passed in there.

Substantively, the only change to this class has been to have the ontModel passed in as part of the check, rather than to rely on one having been set in the constructor.

Off-hand I can't remember specifically why this change was required, but it will be something to do with needing to move where the construction takes place.

Reply:
I understand that you can't be responsible for the shortcomings of any code that you touch. But you created this method signature. It should not be a challenge for you to say what it does.